package/system/procd/patches/0001-ubus-add-bootup-time-measurement.patch

```diff
@@ -0,0 +1,124 @@
+From 6a0d1089a4de97364087405802c53dcd87238947 Mon Sep 17 00:00:00 2001
+From: Florian Eckert <fe@dev.tdt.de>
+Date: Fri, 21 Oct 2022 13:08:38 +0200
+Subject: [PATCH] ubus: add bootup time measurement
+
+Until now it was not possible to determine the booting time. This change
+adds two new elements to the 'ubus system info'.
+
+booting:
+This flag indicates whether the system is still booting.
+
+bootime:
+This value shows in seconds how long the system took until the procd
```

Comment:
> This value shows in seconds

It's too coarse, increase of 1s is a lot of time during boot process, I would go with milliseconds.